tutorials/send-emails-with-serverless-and-tem/index.mdx

```diff
@@ -0,0 +1,155 @@
+  - You have a domain name
+  - You have [downloaded and installed Terraform](https://developer.hashicorp.com/terraform/downloads)
+</Message>
+
+## Clone the GitHub repository
+
+We have created a [GitHub repository](https://github.com/quantumsheep/scw-demo-tem) that contains all the code you need to deploy your infrastructure. Open a terminal and clone the repository to your local machine using the following command:
+
+```shell
+git clone https://github.com/quantumsheep/scw-demo-tem
+```
+
+### Files and directories
+
+In the `scw-tem-demo` directory, you will find three folders: `infra`, `signup-handler` and `super-product`.
+
+- The `infra` folder holds `main.tf`, the Terraform configuration file used to deploy your infrastructure.
+- The `signup-handler` folder contains the files that will configure your function. The `main.go` file is the entry point for your function.
+- The `super-product` folder holds an `index.html.tftpl` file which is a Terraform template. This file contains an HTML template for the registration page of the website we are creating. This template allows you to automatically create and configure the SPF, MX and DKIM records needed for your domain, as well as a Serverless Function.
+
+<Message type="tip">
+  If you want to use a specific function, you can replace `${DISPATCHER_ENDPOINT}` (line 74) in the `index.html.tftpl` file with your function's endpoint.
+</Message>
+
+
+## Build and deploy your infrastructure
+
+Open a terminal and go to the `scw-tem-demo` directory using the following command:
+
+```shell
+cd scw-tem-demo/infra
+```
+
+Run the following command to download the dependencies and initialize Terraform:
+
+`terraform init`
+
+The following output displays:
+
+```shell
+Initializing the backend...
+
+Initializing provider plugins...
+- Finding latest version of hashicorp/local...
+- Finding latest version of hashicorp/archive...
+- Finding latest version of scaleway/scaleway...
+- Installing hashicorp/local v2.4.0...
+- Installed hashicorp/local v2.4.0 (signed by HashiCorp)
+- Installing hashicorp/archive v2.3.0...
+- Installed hashicorp/archive v2.3.0 (signed by HashiCorp)
+- Installing scaleway/scaleway v2.16.3...
+- Installed scaleway/scaleway v2.16.3 (signed by a HashiCorp partner, key ID F5BF26CADF6F9614)
+
+Partner and community providers are signed by their developers.
+If you'd like to know more about provider signing, you can read about it here:
+https://www.terraform.io/docs/cli/plugins/signing.html
+
+Terraform has created a lock file .terraform.lock.hcl to record the provider
+selections it made above. Include this file in your version control repository
+so that Terraform can guarantee to make the same selections by default when
+you run "terraform init" in the future.
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
```

Comment:
```suggestion
You may now begin working with Terraform. Try running `terraform plan` to see
```

Reply:
This is part of the output, not sure I can edit it